app/controllers/webhooks_controller.rb

```diff
@@ -48,6 +48,23 @@ def mailjet
     success
   end
 
+  def mailpace
```

Comment:
This is missing any webhook verification, but I believe we can merge it now and improve this later.